server/src/main/java/org/opensearch/env/NodeEnvironment.java

```diff
@@ -171,18 +172,21 @@ Path resolve(String uuid) {
 
         @Override
         public String toString() {
-            return "NodePath{"
```

Comment:
Please revert this change, no need to use SB here

Reply:
@reta It is not just a change from String concat. to SB. There is a new info added into output:
```java
      .append(", fileCachePath=")
      .append(fileCachePath)
```
also notice that NodeEnvironment already uses SB on lines 428 for similar case. I think it is more elegant if the code sticks to the message building style fashion.

Reply:
Please add  new info to the existing method, no need for SB

Reply:
Done.